src/Middleware/RequestDecompression/src/Microsoft.AspNetCore.RequestDecompression.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core middleware for HTTP Request decompression.</Description>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
```

Comment:
Surprised this is going immediately into Microsoft.AspNetCore.App. Is this line correct❔

Reply:
This is all copied from response compression. As far as I know the only things we exclude from .App are ones with external dependencies.

If we think the new dll is redundant we could put the middleware in with ResponseCompression.